### ISSUE TITLE
automatic test runner for python

### ISSUE THREAD

**Issue by alirezaghey** [exercism/python#1911]
Hi,

I'd like to create a bare-bones automatic test runner for python according to this [description](https://github.com/exercism/automated-tests).

I haven't seen any repository made for this purposes and also didn't find any related open issue, so went ahead and opened one myself.

I've already made [one for JavaScript which is still under review](https://github.com/exercism/javascript-test-runner/pull/1).

Basically, it's a minimal Docker with appropriate bash files to run the tests and also test the whole setup. The output will be pytest's default, since the format is still not decided.

Let me know.
Thank you

**Comment by cmccandless** [comment]
Glad to have some interest in this!

@iHiD, I'm guessing this will need its own repository?

**Comment by iHiD** [comment]
You're both in a team with access to https://github.com/exercism/python-test-runner :)
